plugins/com.google.cloud.tools.eclipse.appengine.compat.test/src/com/google/cloud/tools/eclipse/appengine/compat/GpeMigratorTest.java

```diff
@@ -105,9 +98,7 @@ public void testRemoveGpeRuntimeAndFacets_doNotLogIfMetadataFileDoesNotExist()
     assertFalse(metadataFile.exists());
 
     IFacetedProject facetedProject = ProjectFacetsManager.create(gpeProject);
-    Logger logger = mock(Logger.class);
-    GpeMigrator.removeGpeRuntimeAndFacets(facetedProject, logger);
-    verify(logger, never()).log(any(Level.class), anyString(), any(Throwable.class));
```

Comment:
I feel the test method should be renamed to something like `_doesNotErrorOutIfMetadataFileDoesNotExist()`.